pandas/core/generic.py

```diff
@@ -8930,7 +8930,8 @@ def describe_1d(data):
                 if name not in names:
                     names.append(name)
 
-        d = pd.concat(ldesc, join_axes=pd.Index([names]), axis=1)
+        d = pd.concat([x.reindex(names) for x in ldesc], axis=1,
```

Comment:
use the ``copy=False`` on . the ``.reindex``. remove copy= from the ``pd.concat`` it is not respected misleading here.